src/agni/server/receiver/LoginReceiver.java

```diff
@@ -55,4 +55,8 @@ public void receiveMessage(String ip, ByteBuffer message) {
         notifyLoginRequest(ip, parsedMessage[0], parsedMessage[1]);  
     }
 
+    private void notifyNewUserRequest() {
+        
+    }
+
```

Comment:
This will need to be dealt with as more than a stub since the receiver's have already been implemented. In other words, adding this new functionality to LoginReceiver will need to be done in this PR. I suggest talking to James about this since he implemented the server receivers.